File: src/main/java/com/dayo/testgame/TestGame.java
package com.dayo.testgame;

import com.dayo.simplegameapi.data.GameManager;
import com.dayo.simplegameapi.data.RoomInfo;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.bukkit.plugin.java.JavaPlugin;

public final class TestGame extends JavaPlugin {

    public static TestGame instance;
    @Override
    public void onEnable() {
        instance = this;
        GameManager.Companion.registerGame(new G(), 1);
        // Plugin startup logic

    }

    @Override
    public void onDisable() {
        // Plugin shutdown logic
    }

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if(command.getName().equals("j")) {
            GameManager.Companion.joinPlayer(((Player)sender).getUniqueId(), new RoomInfo(0, 0));
        }
        else if(command.getName().equals("l")) {
            GameManager.Companion.leftPlayer(((Player)sender).getUniqueId());
        }
        return true;
    }
}

File: src/main/java/com/dayo/testgame/TestGame.kt
package com.dayo.testgame

import me.ddayo.simplegameapi.data.GameManager.Companion.registerGame
import me.ddayo.simplegameapi.data.GameManager.Companion.joinPlayer
import me.ddayo.simplegameapi.data.GameManager.Companion.leftPlayer
import me.ddayo.simplegameapi.api.Game
import me.ddayo.simplegameapi.data.RoomInfo
import java.util.UUID
import com.dayo.testgame.TestGame
import java.lang.Runnable
import org.bukkit.plugin.java.JavaPlugin
import com.dayo.testgame.G
import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class TestGame : JavaPlugin() {
    override fun onEnable() {
        instance = this
        registerGame(G(), 2)
        // Plugin startup logic
    }

    override fun onDisable() {
        // Plugin shutdown logic
    }

    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        when (command.name) {
            "j" -> {
                joinPlayer((sender as Player).uniqueId, RoomInfo(0, args[0].toInt()))
            }
            "l" -> {
                leftPlayer((sender as Player).uniqueId)
            }
            "s" -> {
                sender.sendMessage("stop")
                stop = true
            }
        }
        return true
    }

    companion object {
        var instance: TestGame? = null
        var stop = false
    }
}